data/testdata/all_types.golden.json

```diff
@@ -69,8 +69,8 @@
         "type": "number",
         "typeInfo": { "frame": "float64", "nullable": true }
       },
-      { "name": "bool_values", "type": "bool", "typeInfo": { "frame": "bool" } },
-      { "name": "nullable_bool_values", "type": "bool", "typeInfo": { "frame": "bool", "nullable": true } },
+      { "name": "bool_values", "type": "boolean", "typeInfo": { "frame": "bool" } },
```

Comment:
```suggestion
      { "name": "bool_values", "type": "boolean" },
```
In another issue, we should figure out how to avoid writing implicit typeInfo when not necessary.  We just need to pick if things should be nullable by default or not (I think they likely should be)